src/App.tsx

```diff
@@ -10,33 +12,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
+  const tabSelectHandle = (tab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelect = (tab: Tab) => {
```
better to start from `handle`